docs/how-to/code/install-slice/rockcraft.yaml

```diff
@@ -15,4 +15,4 @@ parts:
         source: chisel-releases/
         source-type: local
         override-build:
-            chisel cut --release ./ --root ${CRAFT_PART_INSTALL} openssl_bins
+            /snap/rockcraft/current/bin/chisel cut --release ./ --root ${CRAFT_PART_INSTALL} openssl_bins
```

Comment:
this might mean we want chisel in libexec, @tigarmo, thoughts?

Reply:
err probably? Is that preferable to adding `$SNAP/bin` to $PATH?

Reply:
we talked about this, and went with libexec